build-system/amp-cors.js

```diff
@@ -21,8 +21,8 @@
  * @type {RegExp}
  */
 const ORIGIN_REGEX = new RegExp(
-  '^http://localhost:8000|' +
-    '^http://.+.localhost:8000|' +
+  '^https?://localhost:8000|' +
```

Comment:
nit: this regex could be shorter:

```js
'^https?://(' +
  'localhost:8000|' +
  '.+.(localhost:8000|herokuapp.com))'
```

Reply:
That feels less clear to me. With regexes I try to prefer extreme clarity over removing redundancy

Reply:
I was thinking about bundle size and totally overlooked that this is in `build-system`. Completely agree with you.